pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1670,6 +1670,19 @@
         "4.0.1.0": "4.6.0"
       }
     },
+    "System.Drawing.Extensions": {
```

Comment:
Naming....sigh.  We now have Primitives, Common, and Extensions.  It would be good to be sure we understand the reasoning and purpose behind each of these and document that somewhere (maybe even in the package descriptions, so that people using these can understand the difference).